dockerfile_functions.sh

```diff
@@ -206,6 +225,20 @@ RUN microdnf install openssl curl ca-certificates fontconfig glibc-langpack-en g
 EOI
 }
 
+# Select the CentOS packages.
+print_centos_pkg() {
+	cat >> $1 <<'EOI'
+RUN yum install -y openssl curl ca-certificates fontconfig gzip locales tar \
```

Comment:
please remove `locales` - there is no such package :
```
[root@b4af0547e99e /]# yum install -y openssl curl ca-certificates fontconfig gzip locales tar
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: centos2.hti.pl
 * extras: centos2.hti.pl
 * updates: centos2.hti.pl
base                                                                                                                         | 3.6 kB  00:00:00     
extras                                                                                                                       | 2.9 kB  00:00:00     
updates                                                                                                                      | 2.9 kB  00:00:00     
(1/4): extras/7/x86_64/primary_db                                                                                            | 153 kB  00:00:00     
(2/4): base/7/x86_64/group_gz                                                                                                | 165 kB  00:00:00     
(3/4): base/7/x86_64/primary_db                                                                                              | 6.0 MB  00:00:03     
(4/4): updates/7/x86_64/primary_db                                                                                           | 5.9 MB  00:00:06     
Package gzip-1.5-10.el7.x86_64 already installed and latest version
No package locales available.

```

Reply:
@grzesuav My bad did not realize `locales` did not exist in CentOS or ClefOS. 

One question I had was in the other Docker images, except UBI, you set the language. https://github.com/AdoptOpenJDK/openjdk-docker/blob/921279ff377f2e3e0d5b72fb8039bc97b9939ff2/11/jdk/ubuntu/Dockerfile.hotspot.nightly.slim#L27 But in the UBI images you do not, all that is done is set environment variables. https://github.com/AdoptOpenJDK/openjdk-docker/blob/921279ff377f2e3e0d5b72fb8039bc97b9939ff2/11/jdk/ubi/Dockerfile.hotspot.nightly.full#L22

So, I am not 100% sure on what I need to do for CentOS and ClefOS. Do I **just** need to put the environment variables there? Or do I also need to set the language? If so, should that also be done in the UBI images?

Reply:
not sure actually. UBI is based on which distribution ? RHEL ? I would search in google how to do it to be honest ;)

Reply:
UBI are based on RHEL. I'll see what I can do for CentOS and ClefOS. Leave adding language support for UBI in another PR.

Reply:
Sure, please just remove `locales` as for now they just pretending they are there